etm/operator-interface-spec/etm-oper-api.yaml

```diff
@@ -22,6 +22,8 @@ security:
 tags:
```

Comment:
Constraints for elevated privileges
I recommend deleting entire sectoin.  ETM can only build a small set now